ush/config_defaults.yaml

```
@@ -1004,7 +987,11 @@ nco:
   #-----------------------------------------------------------------------
   #
   # Set variables that are only used in NCO mode (i.e. when RUN_ENVIR is 
-  # set to "nco").  Definitions:
+  # set to "nco"). All variables have the suffix [_dfv] meaning the default value.
+  # This is bacuase they are supposed to be defined in job cards for the 
```

because rather than bacuase.

```suggestion
  # This is because they are supposed to be defined in job cards for the 
```

---

@MichaelLueken, Oops! silly mistake! Thank you for letting me know. Fixed.